packages/core/src/components/ui/ProductGallery/ProductGallery.tsx

```diff
@@ -115,13 +130,32 @@ function ProductGallery({
         data-fs-product-listing-content-grid
         data-fs-content="product-gallery"
       >
-        <div data-fs-product-listing-filters>
-          <FilterSkeleton loading={!hasFacetsLoaded}>
-            {hasFacetsLoaded && facets?.length > 0 && (
-              <Filter facets={facets} filter={filter} />
-            )}
-          </FilterSkeleton>
-        </div>
+        {isDesktop && (
+          <div data-fs-product-listing-filters>
+            <FilterSkeleton loading={!hasFacetsLoaded}>
+              {hasFacetsLoaded && facets?.length > 0 && (
+                <div className="hidden-mobile">
+                  <FilterDesktop.Component
+                    {...FilterDesktop.props}
+                    {...filter}
+                    title={filterCmsData?.title}
+                  />
+                </div>
+              )}
+            </FilterSkeleton>
+          </div>
+        )}
+        {!isDesktop && displayFilter && (
```

Comment:
Just to confirm that `!isDesktop` here will work for both tablet and mobile devices.

Reply:
yes, we can test in the preview link https://sfj-d705510--starter.preview.vtex.app/office
that's because the filter assumes the mobile/tablet version when `!isDesktop`